repl/Main.hs

```diff
@@ -95,7 +95,8 @@ mainWithOptions opts =
           mn <- moduleNameFromPath file
           ctx <- mkInitContext mn
           void $ execExpand initializeKernel ctx
-          st <- execExpand (visit mn >> getState) ctx
+          st <- flip execExpand ctx $ do
```

Comment:
Two questions:
1. Should there be a `completely` here?
2. It seems like this `do` is unnecessary - why not either get rid of it, or replace the `>>` with a line break?

Reply:
Fixed in #38: https://github.com/gelisam/klister/pull/38/files#diff-98af7d285f36c701c8fdfea83eb20dc3R98-R100